### PR TITLE
fix: add adoption date to the charter

### PR DESCRIPTION
**Pull request by fmvilas**
**Description**

Sets the adoption date.

**Related issue(s)**

Required by LF. @tbenzies please have a look. The charter you sent me looks way outdated compared to the agreed one (this one here).

### CODE REVIEW

**Review by derberg — APPROVED** [review]

File: CHARTER.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <p align="center">
   for</br>
   AsyncAPI Project a Series of LF Projects, LLC</br>
-  Adopted ___________
+  Adopted 03-19-2021
```
Comment (derberg) [inline CHARTER.md:5]: shouldn't it be 23.03? the merge date?

Comment (fmvilas) [reply CHARTER.md:5]: I don't know, I'm taking the one they sent me. I think this is the date when I signed the papers IIRC.

**Comment by fmvilas** [comment]
@derberg if you're ok with this, I'm gonna merge it.

**Review by derberg — APPROVED** [review]
🚀

**Pull request merged**